tianshou/env/gym_wrappers.py

```diff
@@ -0,0 +1,33 @@
+import gym
+import numpy as np
+
+
+class ContinuousToDiscrete(gym.ActionWrapper):
+    """Gym environment wrapper to take discrete action in a continous environment.
+
+    Args:
+        env (gym.Environment): gym envirionment with continous action space.
+        action_per_branch (int): number of discrete actions in each dimension
+        of the action space.
+
+    """
+
+    def __init__(self, env: gym.Env, action_per_branch: int) -> None:
+        super().__init__(env)
+        self.action_per_branch = action_per_branch
+        low = env.action_space.low
```

Comment:
There are some mypy errors [here](https://github.com/thu-ml/tianshou/runs/6435253086?check_suite_focus=true). I don't know the "right" way to resolve them but a temporary solution is to add `# type: ignore` at the end of the line.